cfgov/unprocessed/apps/rural-or-underserved-tool/js/call-census.js

```diff
@@ -6,18 +6,18 @@ import count from './count';
 /**
  * Call the census.gov API and display an error if warranted.
  * @param {string} address - An address.
- * @param {Object} rural - Rural data via json file.
- * @param {*} cb - Callback to call (unused).
+ * @param {object} ruralCounties - Rural counties for a chosen year
```

Comment:
Trivial: Param is of `{Array}` type. Also, end with a period to fit with grammar of other code comments.